Sources_v3/StreamChat/Query/UserListQuery.swift

```diff
@@ -75,6 +75,9 @@ public struct UserListQuery<ExtraData: UserExtraData>: Encodable {
     /// Query options. By default the query options contain `presence`.
     var options: QueryOptions = [.presence]
     
+    /// Indicates if the query should be observed for new users.
+    var shouldBeObserved = true
+    
```

Comment:
The documentation here is not the best. What about: `If set to true, newly created users in the database are automatically included in the query if they fit the predicate.`